src/mixxxapplication.cpp

```diff
@@ -168,7 +170,23 @@ bool MixxxApplication::notify(QObject* target, QEvent* event) {
     default:
         break;
     }
-    return QApplication::notify(target, event);
+
+    PerformanceTimer time;
+    time.start();
+
+    bool ret = QApplication::notify(target, event);
+
+    if (time.elapsed() > kEventNotifyExecTimeWarningThreshold) {
+        qDebug() << "Processing event type"
```

Comment:
This debug message makes this issue worse because of the extra time it consumes. 
Is it useful to have this active on the user machine, or should this be a compile option for debug builds only?

Reply:
Maybe with CmdlineArgs::Instance().getDeveloper() ?

Reply:
Done